### PR TITLE
apache: Update to 2.4.34

### PR DESCRIPTION
**Pull request by neheb**
Fixes at least:

CVE-2017-15710
CVE-2017-15715
CVE-2018-1283
CVE-2018-1312

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @heil 
Compile tested: ipq806x

### CODE REVIEW

File: net/apache/Makefile [hunk 1]
```diff
@@ -8,21 +8,21 @@
...
 PKG_SOURCE_NAME:=httpd
 PKG_MAINTAINER:=Thomas Heil <heil@terminal-consulting.de>
 PKG_LICENSE:=Apache License
 
 PKG_SOURCE:=$(PKG_SOURCE_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=@APACHE/httpd/
-PKG_HASH:=c1197a3a62a4ab5c584ab89b249af38cf28b4adee9c0106b62999fd29f920666
+PKG_HASH:=fa53c95631febb08a9de41fd2864cfff815cf62d9306723ab0d4b8d7aa1638f0
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_SOURCE_NAME)-$(PKG_VERSION)
```
Comment (diizzyy) [inline net/apache/Makefile]: PKG_BUILD_DIR is unnecessary?

Comment (neheb) [reply net/apache/Makefile]: eeehhhh. I could remove it but i'd be replacing it in two places. the package is also not mine.

And unfortunately there's a really annoying build error that I haven't figured out yet.

Comment (diizzyy) [reply net/apache/Makefile]: Ahh, it adds version.... Yeah, it should be kept.
There are several reports of mod_proxy being borked btw

**Comment by neheb** [comment]
@heil Updated to 2.4.37. Added -Wno-error to fix compile. Should be redundant once https://github.com/openwrt/packages/pull/7290 is merged.

**Comment by heil** [comment]
thx

**Comment by neheb** [comment]
Check https://github.com/openwrt/packages/pull/7403

**Pull request merged**